src/azul/queues.py

```diff
@@ -392,11 +392,14 @@ def functions_by_queue(self) -> Mapping[str, str]:
         by the queue. The keys and values are fully qualified resource names.
         """
         indexer = load_app_module('indexer', unit_test=True)
-        return {
+        functions_by_queue_ = {
```

Comment:
I don't understand why you [keep adding the underscore](https://github.com/DataBiosphere/azul/pull/4339#discussion_r988449510). There is no conflict here, methods are in a scope separate from that of their local variables.